sdks/python/tox.ini

```diff
@@ -307,3 +331,10 @@ commands =
   coverage report --skip-covered
   # Generate report in xml format
   coverage xml
+
+[testenv:py37-interactive]
```

Comment:
Does this add any additional test coverage? I'd rather fold any 'interactive' tests into the py37 target if possible. Or at least not repeat all the py37 unit tests.

Reply:
I think this addition is actually a rebase error on my part.  These tests were not intentionally added by me.  IIRC, they were added to tox on master then removed in a later commit, and I must've messed up on one of my rebases and left it in.